### ISSUE TITLE
Is ClsVotingTester neccessary in ModelNet40 classification

### ISSUE THREAD

**Issue by leafyseay** [Pointcept/Pointcept#288]
```
# tester
test = dict(type="ClsVotingTester", num_repeat=30)
```
I noticed you used the above test code in [cls-ptv3-v1m1-0-base.py](https://github.com/Pointcept/Pointcept/blob/main/configs/modelnet40/cls-ptv3-v1m1-0-base.py). Does this mean you're using Test Time Augmentation (TTA)? Without this code, the test accuracy is around 0.925, but with ClsVotingTester, it goes up to 0.935. However, the accuracy reported in the PTv2 paper is 0.942. Is the difference made up by using ClsVotingTester?

**Comment by Gofinge** [comment]
I haven't verified it; I just refer to some other works. Actually, I don't like it, a kind of overfitting. Although I make Pointcept support object classification, but I don't have time left to redesign the whole pipeline to make it stronger. I think a very good pre-training is what really needed.
